devicon.json

```
@@ -3310,6 +3310,33 @@
         "color": "#2D333A",
         "aliases": []
     },
+    {
+        "name": "socketio",
+        "tags": [
+            "library",
+            "networking",
+            "websockets"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
```

These versions don't exist so don't put them here.
```suggestion
                "original-wordmark"
```